src/prommis/uky/uky_flowsheet.py

```diff
@@ -862,5 +874,782 @@ def display_results(m):
+    )
+    m.fs.R_gas_scrubber.gas_rate.fix()
+    m.fs.R_gas_scrubber.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": R_gas_scrubber_accounts,
+            "scaled_param": m.fs.R_gas_scrubber.gas_rate,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 3.5 is UKy Roasting - Spray Chamber Quencher (7000-60000 ft**3/min)
+    R_spray_chamber_quencher_accounts = ["3.5"]
+    m.fs.R_spray_chamber_quencher = UnitModelBlock()
+    m.fs.R_spray_chamber_quencher.gas_rate = Var(
+        initialize=11500, units=units.ft**3 / units.min
+    )
+    m.fs.R_spray_chamber_quencher.gas_rate.fix()
+    m.fs.R_spray_chamber_quencher.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": R_spray_chamber_quencher_accounts,
+            "scaled_param": m.fs.R_spray_chamber_quencher.gas_rate,
+            "source": 1,
+            "n_equip": 3,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 3.7 is UKy Roasting - Chiller
+    R_chiller_accounts = ["3.7"]
+    m.fs.R_chiller = UnitModelBlock()
+    m.fs.R_chiller.duty = Var(initialize=131, units=units.MBTU / units.hr)
+    m.fs.R_chiller.duty.fix()
+    m.fs.R_chiller.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": R_chiller_accounts,
+            "scaled_param": m.fs.R_chiller.duty,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # TODO: How should this be handled considering there are multiple feeds with varying concentrations?
+    m.fs.feed_input = Var(initialize=500, units=units.ton / units.hr)
+    m.fs.feed_grade = Var(initialize=356.64, units=units.ppm)
+
+    hours_per_shift = 8
+    shifts_per_day = 3
+    operating_days_per_year = 336
+
+    # for convenience
+    m.fs.annual_operating_hours = Param(
+        initialize=hours_per_shift * shifts_per_day * operating_days_per_year,
+        mutable=False,
+        units=units.hours / units.a,
+    )
+
+    # TODO: Replace this number with the sum of flowsheet.fs.roaster.flow_mol_comp_product (need to convert units)
+    m.fs.recovery_rate_per_year = Var(
+        initialize=39.3
+        * units.kg
+        / units.hr
+        * 0.8025  # TREO (total rare earth oxide), 80.25% REE in REO
+        * m.fs.annual_operating_hours,
+        units=units.kg / units.yr,
+    )
```

Comment:
This seems like the recovery rate of REEs from the plant as a whole. Should the 39.3 be replaced by the mass flow of the product stream from the roaster unit?
![image](https://github.com/prommis/prommis/assets/96305519/45937b83-c0a3-4dab-96a7-e7373b61ff0e)